### DISCUSSION TITLE
Welcome to imap Discussions!

### DISCUSSION THREAD

**Discussion by daohu527** [daohu527/imap#113] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DumplingGood** [comment]
Hi，daohu，
I encountered a question about how to convert traffic lights and pedestrian crossings. I would like to know if the imap box version 0.1.9 supports the conversion of traffic lights and how to convert them. The specific questions are as follows:
1. When visualizing with sim_map, traffic lights cannot be seen; Do I need to set the location in the IMAP source code? The original. xodr file contains traffic light information.
![4901801e-ec9f-4cce-a102-c48955c483ba](https://github.com/user-attachments/assets/633b3753-2eca-4f03-9f5a-5d92543ec6d5)

2. Some stop lines are also not displayed. Do I need to implement the content of some element fields myself?
3. If i want to convert pedestrian crossings, is there a corresponding interface currently available, but it hasn't been implemented yet?

Thank you very much!

**Comment by daohu527** [comment]
@DumplingGood 
When visualizing with sim_map, traffic lights cannot be seen; Do I need to set the location in the IMAP source code? The original. xodr file contains traffic light information.

> yes the traffic lights is not ready for imap, I got it partially working, but not quite right, and I think I need to start over.

Some stop lines are also not displayed. Do I need to implement the content of some element fields myself?

> stop lines are always invisible, but imap has added them. I forget if there is an option to show stop lines on the map, but it is not supported by default 

If i want to convert pedestrian crossings, is there a corresponding interface currently available, but it hasn't been implemented yet?

> Pedestrian crossings are possible, I may need to re-read the spec
